src/pages/Payment/Budget/ApprovedMonthlyBudgets.jsx

```
@@ -1,196 +1,314 @@
+                        .map((m) => {
+                          return m.amount;
+                        })
+                        .reduce(
+                          (previousValue, currentValue) =>
+                            previousValue + currentValue,
+                          0
+                        );
+
+                      if (proj > 0) {
+                        return (
+                          <td>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)